docs/antora/modules/ROOT/pages/management/source.adoc

```diff
@@ -1,120 +1,418 @@
+└----------------------------------------------------------------------------------------------------------------------------------------------------------------------------┘
 ----
+====
+
+=== List
 
-== Status
+View the status of different microservices of Lisk Service in the list of PM2 processes:
 
 [source,bash]
 ----
 pm2 list
 ----
 
-== Stop
+.Response
+[%collapsible]
+====
+.Example output
+[source,bash]
+----
+┌----------------------------------------------------------------------------------------------------------------------------------------------------------------------------┐
+│ id  │ name                                    │ namespace   │ version │ mode    │ pid      │ uptime │ ↺    │ status    │ cpu      │ mem      │ user     │ watching │
+├----------------------------------------------------------------------------------------------------------------------------------------------------------------------------┤
+│ 1   │ lisk-service-blockchain-app-registry    │ default     │ 0.7.0-… │ fork    │ 33328    │ 0s     │ 16   │ online    │ 0%       │ 65.5mb   │ XYZ      │ disabled │
+│ 2   │ lisk-service-blockchain-connector       │ default     │ 0.7.0-… │ fork    │ 33331    │ 0s     │ 0    │ online    │ 0%       │ 63.8mb   │ XYZ      │ disabled │
+│ 4   │ lisk-service-blockchain-coordinator     │ default     │ 0.7.0-… │ fork    │ 33341    │ 0s     │ 0    │ online    │ 0%       │ 54.1mb   │ XYZ      │ disabled │
+│ 3   │ lisk-service-blockchain-indexer         │ default     │ 0.7.0-… │ fork    │ 33338    │ 0s     │ 16   │ online    │ 0%       │ 54.3mb   │ XYZ      │ disabled │
+│ 8   │ lisk-service-export                     │ default     │ 0.7.0-… │ fork    │ 33355    │ 0s     │ 15   │ online    │ 0%       │ 14.1mb   │ XYZ      │ disabled │
+│ 5   │ lisk-service-fee-estimator              │ default     │ 0.7.0-… │ fork    │ 33346    │ 0s     │ 15   │ online    │ 0%       │ 46.9mb   │ XYZ      │ disabled │
+│ 0   │ lisk-service-gateway                    │ default     │ 0.7.0-… │ fork    │ 33326    │ 0s     │ 15   │ online    │ 0%       │ 65.6mb   │ XYZ      │ disabled │
+│ 7   │ lisk-service-market                     │ default     │ 0.7.0-… │ fork    │ 33351    │ 0s     │ 15   │ online    │ 0%       │ 36.9mb   │ XYZ      │ disabled │
+│ 6   │ lisk-service-transaction-statistics     │ default     │ 0.7.0-… │ fork    │ 33348    │ 0s     │ 15   │ online    │ 0%       │ 43.5mb   │ XYZ      │ disabled │
+└----------------------------------------------------------------------------------------------------------------------------------------------------------------------------┘
+----
+====
+
+=== Logs
+To check the logs for the different microservices of Lisk Service, use the command `pm2 logs PROCESS_NAME`, where `PROCESS_NAME` is the respective PM2 managed process that contains the logs you wish to view.
```

Comment:
Please feel free to edit the recommendation.
```suggestion
To check the logs for the different microservices of Lisk Service, use the command `pm2 logs PROCESS_NAME`, where `PROCESS_NAME` is the respective PM2 managed process that contains the logs you wish to view.
Alternatively, it is also possible to the PM2 process ID as seen above. For example, `pm2 logs 0` to check the Lisk Service Gateway logs.
```

Reply:
For the recommendation, I would just suggest the following :
"Alternatively, it is also possible to check the PM2 process ID as shown above. For example, `pm2 logs 0` to check the Lisk Service Gateway logs".

Reply:
```suggestion
To check the logs for the different microservices of Lisk Service, use the command `pm2 logs PROCESS_NAME`, where `PROCESS_NAME` is the respective PM2-managed process that contains the logs you wish to view.
Alternatively, it is also possible to check the logs for a specific PM2 process based on its process ID. For example, `pm2 logs 0` can be used to check the Lisk Service Gateway logs.
```